css/custom.css

```diff
@@ -53,6 +53,18 @@ div.social-container {
   margin-left: 0px;
 }
 
+.languagepicker.roundborders.large div:hover{
```

Comment:
I think, it is good style to only use classes and not div or first child.
Reason: In case of a listing, we would not expect the first element to be different.
Also, a class name usually represents what we want it to do. Encoding structure in the css file duplicates knowledge of structure and duplication leads to being forgotten and errors.

In short: I would be 100% happy to merg this if 
- you use a class name for each different hover method (e.g. hover-black)
- Instead of adding a div around the `<a>`, please add a class name you used before.

What do you think, what would you like to do?